### DISCUSSION TITLE
Base Path

### DISCUSSION THREAD

**Discussion by SaadAlajlan** [zinclabs/zinc#879] [Q&A]
Can i configure Zincsearch base path to be used with Nginx 

like:
 location /zincsearch {
proxy_pass http://localhost:4080/zincsearch
}

**Comment by hengfeiyang** [comment]
Yes, you can

```
location /zincsearch {
   proxy_pass http://localhost:4080
}
```

**Reply by SaadAlajlan** [reply]
Sorry, I didn't explain my case .

I have a server where I can access it on the 443 port only so, I'm using nginx as a reverse proxy for many services one of them is Zincsearch and it cannot be at the root directory because we  have another service served as root
![image](https://github.com/zincsearch/zincsearch/assets/38673587/b0166ee9-bba8-4d6b-9a2c-8a1c413c601e)
 as you can see without the base path I'm getting server error from nginx
 
 PS: when I go to /zincsearch it redirects me as u see in the screenshot when i use the config u provided

**Comment by hengfeiyang** [comment]
Can you check the nginx log, what is the 500 error?
